efiction/tests/test_data/storiesef/5/842.txt

```diff
@@ -1,3 +1,3 @@
 This story was originally Windows 1252.
 
-E�s in ipsum oc�rr�ret. Al�i vidiss� �ffic�endi v�m at, �d ridens gr�ec� detraxit vis. Legim�s vol�tp�t �ompr�hensam �um ne, liber fe�gait �ffic�antur q�� an, usu in v��ib�s sal�tat�s. Nec eu justo tritani qu��sti�. V�l amet m�lorum �bhorr�ant �t, v�lit med�ocritat�m �x pro. Mu���s p�ricula ne sea, �am mand�m�s ass�nti�r an, sed �t �q��d�m ornatus.
\ No newline at end of file
+Eôs in ipsum ocûrrëret. Alïi vidissé éfficïendi vïm at, àd ridens gràecï detraxit vis. Legimùs volûtpât çomprèhensam éum ne, liber feùgait ëfficîantur qùî an, usu in vôçibûs salùtatùs. Nec eu justo tritani quâèstiœ. Vël amet màlorum àbhorréant ât, vëlit medïocritatëm éx pro. Muçîùs pêricula ne sea, èam mandàmûs asséntiôr an, sed ët êqûîdêm ornatus.
```

Comment:
I think an oopsie has happened to this Windows 1252 files as all the diacritics have been trashed. Maybe you can revert the change just to this file and hope it doesn't get saved again with the wrong encoding by whatever trashed it?

Reply:
Weirdly, I now notice that GitHub.com is showing me this diff with and without the correct characters, so it could be that this is actually a GitHub problem rather than a problem with the file. But worth checking as I don't think it was part of your changes anyway. 🤔

Reply:
This is actually a purposeful change. When test_chapters_no_text_processing runs, it actually does read all the files in this directory, and this file triggers the too many warnings exception when the encoding is left as Windows 1252. I converted it to UTF-8 to bypass the error. I'm open to other ideas to bypass the error (mocking the open method? refactoring it to automatically attempt other encoding types?) if we need to keep the original encoding.

Reply:
(comments in Slack for better interactivity)